src/uncrustify_types.h

```diff
@@ -184,6 +184,33 @@ struct chunk_t
       return(str.c_str());
    }
 
+
+   // Issue #2984, fill up, if necessary, a copie of the first chars of the text() string
+   const char *text_first_999(char *for_the_copy)
+   {
+      const char *test_it       = str.c_str();
+      size_t     test_it_length = strlen(test_it);
+
+      size_t     truncate_value = uncrustify::options::debug_truncate();
+
+      if (truncate_value != 0)
+      {
+         if (test_it_length > truncate_value)
+         {
+            memset(for_the_copy, 0, 1000);
+            strncpy(for_the_copy, test_it, truncate_value);
+            char *message = strcat(for_the_copy, " ... <The string is truncate>");
```

Comment:
Should be "truncate**d**". This also seems a little wordy, what about just `" ... (truncated)"`?